snapcraft.yaml

```diff
@@ -909,18 +916,33 @@ parts:
       # Mangle the configure a bit
       QEMUARCH="$(uname -m)"
       [ "${QEMUARCH}" = "ppc64le" ] && QEMUARCH="ppc64"
+
+      # Apply patches from Ubuntu sources.
+      QUILT_PATCHES=debian/patches quilt push -a
+
       sed -i "s/^unset target_list$/target_list=\"${QEMUARCH}-softmmu\"/" configure
       sed -i 's#libseccomp_minver=".*#libseccomp_minver="0.0"#g' configure
 
+      # Extract efi-virtio.rom from ipxe-qemu.
```

Comment:
There is also `pxe-virtio.rom` from that same package. Dunno if that's something we might be interested in eventually (not in that PR).